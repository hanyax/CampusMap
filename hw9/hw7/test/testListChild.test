# Load the small test dataset test List Children

LoadGraph test smallLeastCostGraph.tsv

# Test Node that exist in Graph
ListChildren test A
ListChildren test B
ListChildren test C

# Test Node that does not exist in Graph
ListChildren test Z
