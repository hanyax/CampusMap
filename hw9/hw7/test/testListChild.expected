# Load the small test dataset test List Children

loaded graph test

# Test Node that exist in Graph
the children of A in test are: A(0.000) B(0.500) C(1.000) D(0.250)
the children of B in test are: A(0.500) B(0.000) C(0.200) E(0.125) F(0.500)
the children of C in test are: A(1.000) B(0.200) C(0.000) D(0.500) E(0.167)

# Test Node that does not exist in Graph
the children of Z in test are:
